client/app/components/OnboardingGuide/GuideCoverPhotoPage.js

```diff
@@ -46,17 +47,13 @@ const GuideCoverPhotoPage = (props) => {
       ]),
     ]),
 
-    a({ className: css.nextButton, href: pageData.cta }, t('web.admin.onboarding.guide.cover_photo.add_your_own')),
+    a({ className: css.nextButton, href: Routes.admin_look_and_feel_edit_path() }, t('web.admin.onboarding.guide.cover_photo.add_your_own')),
```

Comment:
This is debatable whether **presentation** level components should be aware of Rails routes or not. The tradeoff is the extra work of having to pass down the routes as props from top level vs putting the Rails dependency deep into the component tree. What do you think @Gnito, what should be the guideline here?